MVVM-C Scene.xctemplate/___VARIABLE_sceneIdentifier___Coordinator.swift

```diff
@@ -3,15 +3,15 @@
 //  ___PROJECTNAME___
 //
 //  Created by ___FULLUSERNAME___ on ___DATE___.
-//  Copyright (c) ___YEAR___ ___ORGANIZATIONNAME___. All rights reserved.
```

Comment:
Důvod změny? Xcode 9 například začal odstraňovat háčky a čárky ze jména autora/firmy. Takže bych tam ponechal hezké ASCII znaky. 🙂

Reply:
Já bych pro změnu nejradši vyhodil tyto dva řádky úplně. Zbytečně to klade některým lidem dilema jestli to u převzatých souborů přepisovat na své jméno a název svého projektu, nebo to nechat být.
(ale vím, že by to vyžadovalo úpravu všech šablon)

Reply:
//  Created by ___FULLUSERNAME___ on ___DATE___. 
--> by sme asi mohli odstranit, kvoli tomu co hovoris @zvonicek 

//  Copyright (c) ___YEAR___ ___ORGANIZATIONNAME___. All rights reserved.
--> ale organizaciu by som asi nechal, co myslite? a kludne nechajme (c)

Reply:
Já bych tam nechal to standardní nastavení, protože pak člověk nemusí řešit dilema co nechat a co už je zbytečné. A nebo ty komentáře vyhodit úplně.

Mimochodem do SwiftLintu lze nastavit kontrola a šablona úvodních komentářů, takže bych to vyřešil dohromady s tím.

Reply:
Je to asi na širší diskusi, např. na dalším iOS meetu. Mě dává smysl tyto komentáře vůbec nemít protože dělají víc problémů než užitku, ale rád bych slyšel názor všech ostatních.

Reply:
ani organizaciu nechces nechat?

Reply:
I to bych dal pryč, protože když píšeme kód pro klienty, tak by ten copyright měl být spíš na ně a ne na nás... Probereme dnes na iOS meetu.

Reply:
Jsou tu pořád ty copyright znaky. 😢